client-admin/README.md

```diff
@@ -42,8 +42,13 @@ get very far. Still it can be useful for developing and debugging builds.
 Optionally, you can [register with Facebook](https://developers.facebook.com/docs/development) and get a Facebook App ID
 to use the Facebook auth features.
 
-If you do so, set the FACEBOOK_APP_ID environment variable in the top level `.env` file, or manually pass it in
-when building or running this application.
+If you do so, set the FB_APP_ID environment variable in the top level `.env` file, or manually pass it in
+when building and running this application.
+
+### Twitter Integration
+
+To enable twitter widgets for user authentication, set the ENABLE_TWITTER_WIDGETS environment variable to `true` in the
```

Comment:
It seemed like a district enough concern to warrant a new variable. The client app doesn't get the api creds, but would always embed the Twitter widget script, relying on the server to handle the rest. So rather than send the api creds (or lack thereof) into the client build I thought it would make more sense to introduce this boolean flag. Of course you would still need to provide the api creds to the server.